src/oic/oic/__init__.py

```diff
@@ -258,19 +258,17 @@ def add_token(self, resp):
 }
 
 
-def claims_match(value, claimspec):
-    if claimspec is None:
-        return True
+def claims_match(claim1, claim2):
+    """
+    Is claim1 equal to or present in claim2?
 
-    for key, val in claimspec.items():
-        if key == "value":
-            if value != val:
-                return False
-        elif key == "values":
-            if value not in val:
-                return False
-                # Whether it's essential or not doesn't change anything here
-    return True
+    :param claim1: First claim value
+    :param claim2: Second claim value
+    :return: Boolean
+    """
+    if isinstance(claim2, Iterable):
+        return claim1 in claim2
```

Comment:
I think you need to do `claim1 in claim2.values()` to keep the logic of previous implementation.